Update es.json

Aitonos opened the pull request:
UPDATE AFTER READING YOUR DISCORD MESSAGES; I Start ti understand how it works, I have no IT background so I miss many basic thngs, but I learn fast ( I hope!)
merged and now Ok.

As I say below ( written yesterday) there still many things I have no text to translate.

Please find v3
I've done major changes according page context and logical language/expressions continuity between pages/threads/words There are some texts appearing in the web but still missing from this es.json like FAQ; Privacy and some boxes/dropdown menus.

In V2 I did all the sources part of SignalK but in the updated version is still unchanged in english. Please check if you have it. Except that part which I haven't touch it tonight, this V3 version is far better and much more polished, if you can change it yourself the Sources of SignalK, Please do it, if not, I'll try to figure out how to see again V2 Thanks

<!--
MAKE SURE TO READ THE CONTRIBUTING GUIDE BEFORE CREATING A PR
https://github.com/epicmaxco/vuestic-admin/blob/master/CODE_OF_CONDUCT.md
-->

<!-- Provide a general summary of your changes in the Title above -->
<!-- Keep the title short and descriptive, as it will be used as a commit message -->

## Description

<!-- Describe your changes in detail -->

## Markup:

<!-- Paste your markup here. -->
<details>

```vue
// Your code
```

</details>

## Types of changes

<!-- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

Aitonos committed:
Update es.json

Dear François:
Please find v3
I've done major changes according page context and logical language/expressions continuity between pages/threads/words
There are some texts appearing in the web but still missing from this es.json like FAQ; Privacy and some boxes/dropdown menus.

In V2 I did all the sources part of SignalK but in the updated version is still unchanged in english. Please check if you have it. Except that part which I haven't touch it tonight, this V3 version is far better and much more polished, if you can change it yourself the Sources of SignalK, Please do it, if not, I'll try to figure out how to see again V2
Thanks

Aitonos committed:
Merge branch 'live' into patch-2

xbgmsharp commented on src/i18n/locales/es.json:
```diff
@@ -358,7 +354,7 @@
     "public_timelapse": "Línea del tiempo pública",
     "instagram_handle": "Nick Instagram",
-    "preferred_homepage": "Pagina inicio preferida",
+    "preferred_homepage": "Pagina preferida DEinicio ",
```
tipo?

Aitonos replied:
Yes, it should be "Pagina preferida de inicio"

xbgmsharp commented on src/i18n/locales/es.json:
```diff
@@ -273,21 +273,17 @@
-    "top_logs_by_avg_speed": "Top Logs by Average Speed",
-    "top_logs_by_max_speed": "Top Logs by Max Speed",
-    "top_logs_by_wind_speed": "Top Logs by Max Wind Speed",
-    "top_moorages_by_duration": "Top Moorages by Stay Duration",
```
You can't remove entries

xbgmsharp commented on src/i18n/locales/es.json:
```diff
@@ -130,7 +130,7 @@
+      "title": "Cuaderno de bitácora - buscar de",
       "filter": {
         "title": "Search for log",
         "name": "Filter by name...",
```
you are missing the search placeholder.

xbgmsharp commented:
Hello,
Thanks for your contribution. Much appreciated. 
You seem to be missing some select placeholder lines:
* https://github.com/xbgmsharp/vuestic-postgsail/blob/live/src/i18n/locales/es.json#L136
* https://github.com/xbgmsharp/vuestic-postgsail/blob/live/src/i18n/locales/es.json#L188
* https://github.com/xbgmsharp/vuestic-postgsail/blob/live/src/i18n/locales/es.json#L223
You also remove some entries.
* https://github.com/xbgmsharp/vuestic-postgsail/blob/live/src/i18n/locales/es.json#L283
You don't have to translate it all but you can not add nor delete entries.

xbgmsharp commented on src/i18n/locales/es.json:
```diff
@@ -171,13 +171,13 @@
       "share": "Compartir",
-      "tags": "Etiquetas",
-      "delete": "Delete Log Book"
+      "tags": "Tags",
```
why not Etiquetas?

xbgmsharp commented:
Please don't use find and replace. It can break existing translation.
https://github.com/xbgmsharp/vuestic-postgsail/blob/live/src/i18n/locales/es.json#L548
https://github.com/xbgmsharp/vuestic-postgsail/blob/live/src/i18n/locales/es.json#L549
I edit the key back to the original word.

xbgmsharp committed:
Update es.json

Fix conflict

xbgmsharp committed:
Update es.json

Update the translation based on my comments.
- keep orignal key.
- add the search placeholder.
- add remove entries
- fix tipo

xbgmsharp commented:
fix with e11342beba4c8808caef613837d5f8405dc28262
Thanks for your contribution.
